meta-resin-common/recipes-connectivity/networkmanager/files/NetworkManager.conf

```diff
@@ -3,4 +3,4 @@ dns=default
 plugins=keyfile
 
 [keyfile]
-unmanaged-devices=interface-name:docker*;tun*
+unmanaged-devices=interface-name:docker*;interface-name:tun*;interface-name:resin*;interface-name:veth*;
```

Comment:
I think we should also add p2p here: 

commit fca1b38774860a9c3d85dc0589f75bd4fc25d8f6
Author: Florin Sarbu <florin@resin.io>
Date:   Fri Apr 1 18:54:38 2016 +0300

    connman: Add p2p to NetworkInterfaceBlacklist
    
    We need to make connman not try to manage p2p interfaces. Othwerwise, this conflicts
    with our existing wireless set-up (which implies connection to an AP) - p2p will take
    precedence over the wlan interface so we will not be able to connect to APs.
    
    Signed-off-by: Florin Sarbu <florin@resin.io>